arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -3705,6 +3705,32 @@
       },
       "description": "The instance view of a virtual machine boot diagnostics."
     },
+    "VirtualMachineIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of virtual machine identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id associated with the virtual machine."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
```

Comment:
My only concern with that is, that right now autorest will generate methods that skip those fields, and hardcode the value in the implementation, and when you add one later, it'll then add the parameter to the call. 

Other than the `description` "The identity type" being a smidgen vague, I was also wondering if you want to set a default for `type`.